pkg/operator/operator.go

```diff
@@ -74,8 +74,8 @@ const (
 
 	// The k8s Application, will be exposed as component name.
 	KubernetesAppName    = "app"
-	CollectorAppName     = "managed-prometheus-collector"
-	RuleEvaluatorAppName = "managed-prometheus-rule-evaluator"
+	CollectorAppName     = "managed_prometheus_collector"
```

Comment:
I think your rebase may be stale here, on main this isn't changed: https://github.com/GoogleCloudPlatform/prometheus-engine/blob/main/pkg/operator/operator.go#L77

Reply:
Ah, I resolved the rebase incorrectly.